main.go

```diff
@@ -2,12 +2,17 @@ package main
 
 import (
 	"log"
+	"net/http"
 
+	"github.com/maddevsio/comedian/api"
 	"github.com/maddevsio/comedian/chat"
 	"github.com/maddevsio/comedian/config"
 )
 
 func main() {
+	handler := api.GetHandler()
+	http.ListenAndServe(":8080", handler)
+
```

Comment:
It's not OK. Bot will not start, because it'll block on http.ListenAndServe method. Make it async